vendor/assets/leaflet/leaflet.osm.js

```diff
@@ -64,6 +64,16 @@ L.OSM.HOT = L.OSM.TileLayer.extend({
   }
 });
 
+L.OSM.GPS = L.OSM.TileLayer.extend({
+  options: {
+    url: document.location.protocol === 'https:' ?
+      'https://{s}.gps-tile.openstreetmap.org/lines/{z}/{x}/{y}.png' :
+      'http://{s}.gps-tile.openstreetmap.org/lines/{z}/{x}/{y}.png',
+    maxZoom: 20,
+    subdomains: 'abc'
+  }
+});
+
```

Comment:
@derickr well that builds on this - they're two components of the same things although I agree the split is a bit arbitrary.

@IOOI-SqAR open the upstream PR but there's not need to remove it here as we still need it, we just don't want out version diverging from upstream.